rcljava/src/main/java/org/ros2/rcljava/contexts/Context.java

```diff
@@ -33,10 +33,14 @@
   /**
    * Initialize the context.
    * // TODO(jacobperron): Pass arguments for parsing
```

Comment:
I think this PR resolves this TODO, not the TODO on the next line, which I was thinking about an equivalent to [`rclcpp::InitOptions`](https://github.com/ros2/rclcpp/blob/bb3debcfbaf159794298aa4096bcc04b81ef15ab/rclcpp/include/rclcpp/init_options.hpp#L28).

Reply:
Yeah, I deleted the wrong line :joy:.
Fixed in https://github.com/osrf/ros2_java/pull/45/commits/fb1741cfd137e6a046555fe4bbb946bf4f1bb731.